general-meetings.md

```diff
@@ -1,21 +0,0 @@
-# General meetings
```

Comment:
I have deleted this file as I don't think we have this style of meeting anymore? Happy to be corrected though.